bazel/cas/conn_interfaces/conn_interfaces.go

```diff
@@ -0,0 +1,15 @@
+package conn_interfaces
```

Comment:
rename to connection or something without underscores

Reply:
renamed to connection-api.  didn't like the no underscores option (connectionapi)

Reply:
The directory is now connection-api, but the package name is still connection_api. The golang style guide endorses using single word names (https://golang.org/doc/effective_go.html#names), could we just use "connection"?

Reply:
+1